content/testing/fluent-assertions.md

```diff
@@ -0,0 +1,96 @@
+---
+title: FluentAssertions
+showMetadata: true
+editable: true
+showToc: true
+---
+
+สมมติว่าเรามี Data Structure ดังนี้ คำถามคือเราจะ Assert อย่างไร
+
+```c#
+public class Todo
+{
+    public int id  { get; set; }
+    public string title  { get; set; }
+    public bool completed  { get; set; }
+}
+
+public class TodoList
+{
+
+    public IEnumerable<Todo> GetTodoList()
+    {
+        return new Todo[]
+        {
+            new Todo{
+                id = 0,
+                title = "Shopping",
+                completed = false
+            },
+            new Todo{
+                id = 1,
+                title = "Running",
+                completed = true
+            },
+            new Todo{
+                id = 2,
+                title = "Reading a book",
+                completed = false
+            }
+        };
+    }
+}
+```
+
+ถ้าใช้ xunit อย่างเดียว อาจจะยุ่งยาก ลองใช้ [FluentAssertions](https://www.nuget.org/packages/FluentAssertions)
```

Comment:
Very minor, the name of our testing framework that we use in this example is `xUnit`.

Reply:
Yes, i’ll